lib/timeline/timeline.dart

```diff
@@ -61,13 +73,25 @@ class TimelineScreen extends Screen {
       ..disabled = true
       ..click(_resumeRecording);
 
-    final PButton trackWidgetBuildsButton = new PButton('Track widget builds')
-      ..small();
-    final PButton perfOverlayButton = new PButton('Performance overlay')
-      ..small();
-    final PButton repaintRainbowButton = new PButton('Repaint rainbow')
-      ..small();
-    final PButton debugDrawButton = new PButton('Debug draw')..small();
+    // TODO(kenzie): perhaps add same icons we use in IntelliJ to these buttons.
+    // This would help to build icon familiarity.
+    PButton createButton(String text, String extensionName) {
```

Comment:
nit:
createExtensionButton and move this someowhere outside this class as it looks generally useful.

Reply:
Done.